[2021-07-16 16:40:20,709] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: psotgrressdftest.restart_test 2021-07-16T16:39:37.542918+00:00 [queued]>
[2021-07-16 16:40:20,917] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: psotgrressdftest.restart_test 2021-07-16T16:39:37.542918+00:00 [queued]>
[2021-07-16 16:40:20,917] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-07-16 16:40:20,918] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-07-16 16:40:20,919] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-07-16 16:40:20,938] {taskinstance.py:1087} INFO - Executing <Task(TriggerDagRunOperator): restart_test> on 2021-07-16T16:39:37.542918+00:00
[2021-07-16 16:40:20,952] {standard_task_runner.py:52} INFO - Started process 8810 to run task
[2021-07-16 16:40:20,958] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'psotgrressdftest', 'restart_test', '2021-07-16T16:39:37.542918+00:00', '--job-id', '1176', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/POSTGRESRSERS.py', '--cfg-path', '/tmp/tmpupqx632y', '--error-file', '/tmp/tmp4summv0r']
[2021-07-16 16:40:20,959] {standard_task_runner.py:77} INFO - Job 1176: Subtask restart_test
[2021-07-16 16:40:21,044] {logging_mixin.py:104} INFO - Running <TaskInstance: psotgrressdftest.restart_test 2021-07-16T16:39:37.542918+00:00 [running]> on host 13346afafa5b
[2021-07-16 16:40:21,163] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=psotgrressdftest
AIRFLOW_CTX_TASK_ID=restart_test
AIRFLOW_CTX_EXECUTION_DATE=2021-07-16T16:39:37.542918+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-16T16:39:33.871100+00:00
[2021-07-16 16:40:25,215] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=psotgrressdftest, task_id=restart_test, execution_date=20210716T163937, start_date=20210716T164020, end_date=20210716T164025
[2021-07-16 16:40:25,287] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-16 16:40:25,367] {local_task_job.py:151} INFO - Task exited with return code 0
